content/1.7/back-office/shop-parameters/customer-settings/view-groups.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Groups
+weight: 1
+---
+
+### Call to action
+
+ - Add new group, redirects to the [specs of the add new group]({{<ref "add-new-group.md">}} "Add new group) 
+
+Below an explanation is shown: 
+
+    PrestaShop has three default customer groups:
+    Visitor - All persons without a customer account or customers that are not logged in.
+    Guest - All persons who placed an order through Guest Checkout.
+    Customer - All persons who created an account on this site.
+
+## Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Add new
+
+When pressed redirects to the [specs of the add new group]({{<ref "add-new-group.md">}} "Add new group) 
+
+2. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+3. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+Close: After clicking on it, the modal is closed
+Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+4. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+# Groups 
+
+## Columns 
```

Comment:
Add filters

Reply:
Could you clarify what needs to be fixed here? I've described the filtering (search) and the sorting in the columns @fabienvallon